source/systemUtils.py

```diff
@@ -63,6 +63,45 @@ def hasUiAccess():
 		ctypes.windll.kernel32.CloseHandle(token)
 
 
+TokenOrigin = 17
+
+
+class TOKEN_ORIGIN(ctypes.Structure):
```

Comment:
NVDA's [coding style](https://github.com/nvaccess/nvda/blob/master/devDocs/codingStandards.md#identifier-names) suggests this should be TokenOrigin